### ISSUE TITLE
Bug in first level of TAG2

### ISSUE THREAD

**Issue by Moiras** [nopjne/EntityHero#6]
Hi, first of all thank you for this awesome Editor.

I have been trying to create master levels for the second DLC but in the first level something weird happens. After loading the level in the editor with the meathook interface and saving (even if I didn't change anything) both encounters with the green wolf and the damaging fog start bugging. I start taking damage when I enter the fog area (but the fog is not there) and in the first encounter the wolf just runs to the torch, jumps on in and the dissappears. The encounter never starts and I keep taking damage until I use the noclip code to move out of the fog area or run back to the start of the level.

I have done some testing.
As I said this happens even if I don't change anything (just open from MH>Save>Reload) which makes me think there is an issue with how the program opens the entities file and it doesn't read those encounters correctly (or it doesn't even recognize them) and after saving and reloading the level with part of those encounters corrupted (or missing) this bug happens.
If I follow the damage to its source with the indicator it is coming from the same spot as the damage in those encounters. And you only take damage when you're inside the area covered in fog (when the encounter works of course). That's how I figured out it was the fog that was bugged.

I have searched a lot but I am pretty new to modding and can't seem to figure out where the problem is. Any help would be appreciated.
Thank you :)

**Comment by Moiras** [comment]
After a little more testing I found that the first wolf sometimes doesn't even jump to the torch he just stands there while you take damage.

In the following video you can see all issues I described:

https://www.youtube.com/watch?v=1MKXsxCIaCY

(Sorry for watermark)

**Comment by nopjne** [comment]
Thanks for reporting. While the aim is to test some of the functionality before commits, no formal automated testing exists yet.
Unfortunately that means things break. 

I checked the issue and it looks like the parser is broken where it omits "=" signs for "properties".

			properties {
					num = 1
					
			properties **=** {
					num = 1**;**

**Comment by Moiras** [comment]
Thank you for the quick response. I tried using an older version as suggested and it did indeed fix the bug. I also noticed another bug that is only present in the last version. Whenever I duplicate an eventCall inside an encounter if I try to duplicate that newly created event the program crashes. This doesn't happen in ver 0.7.


I will countinue to test and work on those levels, I'll report any further bugs I find but aside from a few random crashes I have managed to easily read, understand and modify encounters even though I have literally no modding experience whatsoever.
Great tool, keep up the good work :)
